bertopic/backend/_spacy.py

```diff
@@ -20,7 +20,7 @@ class SpacyBackend(BaseEmbedder):
 
     ```python
     import spacy
-    from bertopic.backend import SpacyBackend
+    from bertopic.backend._spacy import SpacyBackend
```

Comment:
I noticed `SpacyBackend` wasn't actually exported in `bertopic.backend.__init__`, so I think these were small typos since you can't import from the backend namespace directly I don't think?

Reply:
I would actually want to expose them in a future update to present these backends with more options. So you can currently leave them as is otherwise the documentation of all other backends will need to be changed which is a bit out of the scope in this PR.

Reply:
I see - as is, they're not valid docstrings since importing directly would throw an ImportError. I found this because I was trying to use them _as_ documentation for the class :joy: so while they aren't exposed maybe via the __init__ function's export, it is still confusing for contributors who  might read it